test/e2e/horizontal_pod_autoscaling.go

```diff
@@ -24,8 +26,9 @@ import (
 )
 
 const (
-	kind        = "replicationController"
-	subresource = "scale"
+	kind             = "replicationController"
+	subresource      = "scale"
+	stabilityTimeout = 5 * time.Minute
```

Comment:
With the current timeouts it will never downscale before 5 minutes:
https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/podautoscaler/horizontal.go#L48